packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.spec.js

```diff
@@ -5,6 +5,14 @@ import { Tasks } from './Tasks'
 import asyncStates from '@zooniverse/async-states'
 
 describe('Tasks', function () {
+  const tasks = [{
```

Comment:
There's a factory for a test mock of single choice task types if you want to use it. I'd like us to move toward using those so our mocks are consistent across tests. This isn't really a blocker for me though.